### PR TITLE
feat(integrations): added documentation for authorizenet integration

### PR DESCRIPTION
**Pull request by MyuddinKhatri**
added documentation for authorizenet integration

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Also, it will create Payment Gateway Account entry. Payment Gateway Account is configuration hub from this you can set account head from existing COA, default Payment Request email body template.
+
+<img class="screenshot" alt="Payment Gateway Account" src="{{docs_base_url}}/assets/img/setup/integrations/payment_gateway_account_authorizenet.png">
+
+After enabling service and configuring Payment Gateway Account your system is able to accept online payments.
+
+#### Supporting transaction currencies
+
+USD,CAD
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md:30]: Please add a statement to this.

File: erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!-- add-breadcrumbs -->
+#Authorizenet Integration
+
+A payment gateway is an e-commerce application service provider service that authorizes credit card payments for e-businesses, online retailers, bricks and clicks, or traditional brick and mortar.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md:4]: Please change 'service provider service' to 'service provider'

File: erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+To enable Authorizenet payment service, you need to configure parameters like API Login ID and API Transaction Key.
+
+<img class="screenshot" alt="AuthorizeNet Settings" src="{{docs_base_url}}/assets/img/setup/integrations/authorizenet_settings.png">
+
+On enabling service, the system will create Payment Gateway record and Account head in chart of accounts having account type as Bank.
+
+<img class="screenshot" alt="AuthorizeNet Chart Of Accounts" src="{{docs_base_url}}/assets/img/setup/integrations/authorizenet_coa.png">
+
+Also, it will create Payment Gateway Account entry. Payment Gateway Account is configuration hub from this you can set account head from existing COA, default Payment Request email body template.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md:22]: Please reword the sentence. It is not clear.

File: erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+`Explore > Integrations > Authorizenet Settings`
+
+#### Setup Authorizenet
+
+To enable Authorizenet payment service, you need to configure parameters like API Login ID and API Transaction Key.
+
+<img class="screenshot" alt="AuthorizeNet Settings" src="{{docs_base_url}}/assets/img/setup/integrations/authorizenet_settings.png">
+
+On enabling service, the system will create Payment Gateway record and Account head in chart of accounts having account type as Bank.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md:18]: On enabling 'the' service....

Comment (akshitakl) [reply erpnext_documentation/www/docs/user/manual/en/erpnext_integration/authorizenet-integration.md:18]: Please add clarity to the sentence

**Comment by barredterra** [comment]
Please fix spelling:

- organisations -> organizations,
- centralised -> centralized

**Pull request closed without merging**